.eslintrc.js

```diff
@@ -1,45 +1,18 @@
 module.exports = {
-  env: {
-    browser: true,
-    es6: true,
-  },
   extends: [
-    'standard',
-    'standard-react',
-    'prettier/react',
+    'react-app',
+    'prettier/@typescript-eslint',
```

Comment:
These are the ejected defaults via `yarn lint --write-file` so I could use the hooks plugin. Do we want to be explicit here with our own standards?

Reply:
Good question, I've never looked, but do you know if there a good way to diff between different eslint configs?

Since we're currently in a reset phase, this may also be a good time to standardize an eslint config for JS and TS.

Reply:
I was thinking about this and I really like the idea of standardising around a single opinionated tool. The more of this configuration we can offload the more we can focus on the building, so for these TS conversions I was thinking we could just straight up use the provided values and leave it there, see how we get on?

These customisations I made were actually unnecessary, the tool ships with all the setting we would want, my editor just couldn't see the configuration.

I followed their suggested fix https://github.com/formium/tsdx/issues/379#issuecomment-567081159 and exported unedited default settings in https://github.com/aragon/use-viewport/pull/9/commits/9dd0d73ed23c5d51dbe399cf4959dbc70a0bfe34

Reply:
Interesting, so VSC can't automatically resolve TSDX's default config without ejecting them?

I've been procrastinating on setting up "standard" linting configs in https://github.com/aragon/github for _forever_, but now is a good chance. I'll get a structure going—I was mainly thinking of packaging something like `eslint-config-aragon-ts-react` (and etc. for other environments).

Reply:
Having a package for our config would be lovely! And yeah, for some reason VSCode sometimes has a hard time reading configs with TSDX 😅